### PR TITLE
Fix/increases logging and add external logs

### PR DESCRIPTION
**Pull request by jungeunyoon**

### CODE REVIEW

File: src/widget/calendar.js [hunk 1]
```diff
@@ -316,63 +320,101 @@ RiseVision.Calendar = (function (gadgets) {
...
-        if (daysNode) {
-          while (daysNode.firstChild) {
-            fragment.appendChild(daysNode.firstChild);
-          }
+    function setAdditionalParams( params ) {
+      // Load fonts.
+      var fontSettings = [
+        {
+          "class": "date",
+          "fontStyle": params.dateFont
```
Comment (jungeunyoon) [inline src/widget/calendar.js]: @Rise-Vision/content this was issue #117 
We needed to upgrade `widget-common` to add external logging but with it we needed to change `fontSetting` --> `fontStyle`

**Comment by jungeunyoon** [comment]
[widget-google-calendar -- Issue 114 [2]](https://trello.com/c/IoWuTIAy/5028-widget-google-calendar-issue-114-2)

**Comment by rudibravo** [comment]
@jungeunyoon looks good. curious, did you have a chance to test this widget?

**Comment by jungeunyoon** [comment]
@rudibravo yes i tested the staged version (https://apps.risevision.com/editor/workspace/996bf03e-1ae3-4c22-88bb-b815d8aa4fcd/) and the local version as well.

**Comment by rudibravo** [comment]
@jungeunyoon 👍

**Pull request merged**